go.mod

```diff
@@ -79,16 +79,17 @@ require (
 	github.com/tinylib/msgp v1.1.8
 	github.com/twitchtv/twirp v8.1.3+incompatible
 	github.com/urfave/negroni v1.0.0
+	github.com/valyala/fasthttp v1.49.0
```

Comment:
Looks like this dep was already here but just wasn't listed in our go.mod for some reason?

Reply:
It went from having an `// indirect` comment to not. I think this is because we now have a `fasthttp` integration so we directly depend on the library. So dependabot ran `go mod tidy` and picked up that it's now a direct dependency.